infra/variables.tf

```
@@ -18,7 +18,7 @@
 variable "image_version" {
   type        = string
   description = "Version of application image to use"
-  default     = "0.5.0" // x-release-please-version
+  default     = "0.5.0-pr88" // x-release-please-version
```

Reminder to change this back:
```suggestion
  default     = "0.5.0" // x-release-please-version
```

---

I'm going to update this to instead alter the application_image local in main.tf. Changing this value here causes linting issues.